plugins/vli/fu-vli-pd-device.c

```diff
@@ -371,6 +371,7 @@ fu_vli_pd_device_setup(FuDevice *device, GError **error)
 						       FU_VLI_DEVICE_KIND_VL104);
 				break;
 			case 0xA0:
+				printf("vl105 found\n");
```

Comment:
I don't think you want this debug line

Reply:
Thanks for the catch! Would it be easier after corrections to redo the branch for a cleaner commit history?

Reply:
nah, just pile on the commits and we can squash it when all the CI is okay.